### ISSUE TITLE
"sslv3 alert bad certificate" with some servers that require a client cert

### ISSUE THREAD

**Issue by nabla-c0d3** [nabla-c0d3/sslyze#532]
```
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/local/Cellar/python@3.7/3.7.12/Frameworks/Python.framework/Versions/3.7/lib/python3.7/threading.py", line 926, in _bootstrap_inner
    self.run()
  File "/Users/nabla/Documents/tools/sslyze/sslyze/scanner/_mass_connectivity_tester.py", line 108, in run
    network_configuration=scan_request.network_configuration,
  File "/Users/nabla/Documents/tools/sslyze/sslyze/server_connectivity.py", line 112, in check_connectivity_to_server
    server_location=server_location, network_config=network_configuration,
  File "/Users/nabla/Documents/tools/sslyze/sslyze/server_connectivity.py", line 341, in _detect_client_auth_requirement_with_tls_1_3
    ssl_connection_auth.ssl_client.read(1)
  File "/Users/nabla/Documents/tools/sslyze/.venv/lib/python3.7/site-packages/nassl/ssl_client.py", line 236, in read
    decrypted_data = self._ssl.read(size)
nassl._nassl.OpenSSLError: error:14094412:SSL routines:ssl3_read_bytes:sslv3 alert bad certificate
```

**Comment by nabla-c0d3** [comment]
Fixed in https://github.com/nabla-c0d3/nassl/commit/ec800eb7bfba3c5484c9019059f93f07c17c1e95

**Comment by nabla-c0d3** [comment]
Fixed in [v5.0.0](https://github.com/nabla-c0d3/sslyze/releases/tag/5.0.0).
